webviz_config/_docs/_build_docs.py

```diff
@@ -162,3 +162,23 @@ def build_docs(build_directory: pathlib.Path) -> None:
     (build_directory / "sidebar.md").write_text(
         template.render({"packages": plugin_documentation.keys()})
     )
+
+
+def _split_docstring(docstring: str) -> List[str]:
+    """Divide docstring by splitting on ---, also unindent
+    in case of indented docstrings (similar to this one)
+    """
+    lines = docstring.strip().split("\n")
+    if len(lines) == 1:
+        return lines
+
+    indent_spaces = 1000  # far beyond max width
+    for line in lines[1:]:
+        if line.strip() == "":
+            continue
+        indent_spaces = min(indent_spaces, (len(line) - len(line.lstrip())))
+    unindented_lines = [lines[0]] + [
+        line.strip() if line.strip() == "" else line[indent_spaces:]
+        for line in lines[1:]
+    ]
+    return "\n".join(unindented_lines).split("\n---\n")
```

Comment:
Does this work also with tab characters `\t`? It is not considered good Python style to have tabs instead of spaces, but some package authors might still use tabs? Something to support here?

Reply:
Think strip() without arguments should remove any form of whitespace, including tabs, but can add a test for it to be certain

Reply:
Good. Mix is ok not to support 😄

Reply:
Have also added a test with tabs 👍